packages/net-stubbing/lib/external-types.ts

```diff
@@ -254,6 +250,11 @@ export interface RouteMatcherOptionsGeneric<S> extends RouteMatcherCompatOptions
    * If 'false', only HTTP requests will be matched.
    */
   https?: boolean
+  /**
+   * If `true`, will match the supplied `url` against incoming `path`s.
+   * Cannot be set without `url` or with `path`.
```

Comment:
Slightly confusing. 😅 

```suggestion
   * Requires a `url` argument. Cannot be used with a `path` argument.
```